modules/apps/batch-engine/batch-engine-test/src/testIntegration/resources/com/liferay/batch/engine/internal/test/dependencies/batch10/batch10/workflow-content-json-string.txt

```diff
@@ -0,0 +1 @@
WorkflowStatusManagerUtil.updateStatus(WorkflowConstants.getLabelStatus(\"denied\"),workflowContext);","WorkflowStatusManagerUtil.updateStatus(WorkflowConstants.getLabelStatus(\"pending\"),workflowContext);",""]},{"#tag-name":"script-language","#value":"groovy"},{"#tag-name":"execution-type","#value":"onAssignment"}]},{"#tag-name":"notification","#child-nodes":[{"#tag-name":"name","#value":"CreatorModificationNotification"},{"#tag-name":"template","#cdata-value":["Yoursubmissionwasrejectedby${userName},pleasemodifyandresubmit."]},{"#tag-name":"template-language","#value":"freemarker"},{"#tag-name":"notification-type","#value":"email"},{"#tag-name":"notification-type","#value":"user-notification"},{"#tag-name":"recipients","#child-nodes":[{"#tag-name":"user"}],"receptionType":"to"},{"#tag-name":"execution-type","#value":"onAssignment"}]}]},{"#tag-name":"assignments","#child-nodes":[{"#tag-name":"user"}]},{"#tag-name":"labels","#child-nodes":[{"#tag-name":"label","#value":"update","language-id":"en_US"}]},{"#tag-name":"transitions","#child-nodes":[{"#tag-name":"transition","#child-nodes":[{"#tag-name":"labels","#child-nodes":[{"#tag-name":"label","#value":"resubmit","language-id":"en_US"}]},{"#tag-name":"name","#value":"resubmit"},{"#tag-name":"target","#value":"machine-review"},{"#tag-name":"default","#value":"true"}]}]}]},{"#tag-name":"task","#child-nodes":[{"#tag-name":"name","#value":"machine-review"},{"#tag-name":"description","#value":"MachineActionreviewsthisentry."},{"#tag-name":"metadata","#cdata-value":["","{","\"xy\":[","165,","292","]","}",""]},{"#tag-name":"actions","#child-nodes":[{"#tag-name":"action","#child-nodes":[{"#tag-name":"name","#value":"MachineAction"},{"#tag-name":"description"},{"#tag-name":"script"},{"#tag-name":"script-language","#value":"function#liferay-sample-etc-spring-boot-workflow-action-1"},{"#tag-name":"priority","#value":"1"},{"#tag-name":"execution-type","#value":"onEntry"}]}]},{"#tag-name":"assignments","#child-nodes":[{"#tag-name":"user","#child-nodes":[{"#tag-name":"screen-name","#value":"test"}]}]},{"#tag-name":"labels","#child-nodes":[{"#tag-name":"label","#value":"Machinereview","language-id":"en_US"}]},{"#tag-name":"transitions","#child-nodes":[{"#tag-name":"transition","#child-nodes":[{"#tag-name":"labels","#child-nodes":[{"#tag-name":"label","#value":"approve","language-id":"en_US"}]},{"#tag-name":"name","#value":"approve"},{"#tag-name":"target","#value":"approved"},{"#tag-name":"default","#value":"true"}]},{"#tag-name":"transition","#child-nodes":[{"#tag-name":"labels","#child-nodes":[{"#tag-name":"label","#value":"reject","language-id":"en_US"}]},{"#tag-name":"name","#value":"reject"},{"#tag-name":"target","#value":"update"},{"#tag-name":"default","#value":"false"}]}]}]}],"xmlns":"urn:liferay.com:liferay-workflow_7.4.0","xsi:schemaLocation":"urn:liferay.com:liferay-workflow_7.4.0http://www.liferay.com/dtd/liferay-workflow-definition_7_4_0.xsd","xmlns:xsi":"http://www.w3.org/2001/XMLSchema-instance"}
```

Comment:
This file is also very confusing. Why don't you just create a file with the whole Workflow definition?